src/tv2-common/content/dve.ts

```diff
@@ -452,8 +452,16 @@ export function MakeContentDVE2<
 		context.notifyUserWarning(`DVE Graphics Template JSON is not valid for ${dveConfig.DVEName}`)
 	}
 
-	const keyFile = dveConfig.DVEGraphicsKey ? dveConfig.DVEGraphicsKey.toString() : ''
-	const frameFile = dveConfig.DVEGraphicsFrame ? dveConfig.DVEGraphicsFrame.toString() : ''
+	let keyFile = dveConfig.DVEGraphicsKey ? dveConfig.DVEGraphicsKey.toString() : undefined
+	let frameFile = dveConfig.DVEGraphicsFrame ? dveConfig.DVEGraphicsFrame.toString() : undefined
+
+	if (keyFile && config.studio.DVEFolder) {
+		keyFile = `${config.studio.DVEFolder}/${keyFile}`
```

Comment:
What if the user entered a path with a trailing slash and we ended up with two consecutive? Probably nothing bad will happen because most OSes will ignore it, but I'm not sure about all pieces of software in our pipeline.

Reply:
This is a general concern across all configs in the system. Perhaps we want a helper function to concatenated paths or some other high level magic.

Reply:
I've added two functions to handle concatenating these paths